config/routes.rb

```diff
@@ -454,6 +454,15 @@
       get '/remove' => 'provider_users#confirm_remove', as: :confirm_remove_provider_user
       delete '/remove' => 'provider_users#remove', as: :remove_provider_user
     end
+
+    get '/provider-relationship-permissions/:training_provider_id/:ratifying_provider_id/success' => 'provider_relationship_permissions#success',
```

Comment:
Do we need the `:training_provider_id` part of the URL? Only training providers will be able to set permissions. 

It's possible now for a provider to craft a url which allows them to edit another provider's relationships.

Reply:
As discussed we do need the training provider id but to shore up the access to these actions and prevent URL crafting: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2167/commits/3ea2069d629835472b418e95c5d3f5e46d213dd1